### ISSUE TITLE
Unnecessary chunks and requests

### ISSUE THREAD

**Issue by dNitro** [mike-diamond/media-query-splitting-plugin#15]
Unnecessary chunk generated which ultimately results in unnecessary requests. with just a simple desktop style ( i.e. `@media screen and (max-width: 1025px)` ) someone got all others media chunks that are empty and are not needed:
```
.
├── css
│   ├── main.5f62d0506ecd9afde66f.css
│   ├── main.desktop.5f62d0506ecd9afde66f.css
│   ├── main.mobile.5f62d0506ecd9afde66f.css
│   ├── main.tablet.5f62d0506ecd9afde66f.css
│   ├── main.tabletLandscape.5f62d0506ecd9afde66f.css
│   └── main.tabletPortrait.5f62d0506ecd9afde66f.css
└── index.html
```
while it's much better to generate just the related ones:
```
.
├── css
│   ├── main.5f62d0506ecd9afde66f.css
│   └── main.desktop.5f62d0506ecd9afde66f.css
└── index.html
```
It would be very great that someone could determine which chunks he/she wanted to generate. via config: When he does not provide a boundary that media should not be generated at all. So the following config:
```javascript
new MediaQuerySplittingPlugin({
      // This is default config (optional)
      media: {
        tabletLandscapeEnd: 1024,
      },
      minify: true,
      units: 'px',
    })
```
would generate only `desktop` chunk and ignore others.

**Comment by mike-diamond** [comment]
Will fix in the next release, already work on it

**Comment by masi** [comment]
Would love that!
IMHO the performance penalty for the mobile queries are on the desktop less server then the extra CSS for mobile. So at least an option to include mobile in the main chunk would be nice.

Cheers!

**Comment by kilgarenone** [comment]
Sorry, any update on this?

**Comment by erichstark** [comment]
@dNitro it depends.. Sometimes there could be query which is related for tablet (min width 768), but there is no rule for desktop and you need it also.

**Comment by kalyantm** [comment]
@mike-diamond Any update on this? A guide or anything if done? I also have a use case where I just need mobile and desktop styles (dont need the tablet bundle css to be generated at all)

**Comment by mike-diamond** [comment]
Implemented in v 2.0.0
